document-site.html

```diff
@@ -65,6 +65,10 @@ <h2>Components</h2>
                     <li>
                         Navigation</li>
                 </a>
+                <a href="#text-utilities" onclick="hideSideNavBar()">
```

Comment:
Always try to refrain from using inline JS in HTML tags
You can use a regular query selector inside an external JS file and handle the same event

Reply:
Any specific reason to not use inline JS?